### PR TITLE
Update the wiki bazel instructions to add some tips around python and jdk8

### PR DESCRIPTION
**Pull request by masclot**
## Explanation
Update the bazel setup instructions wiki to add some tips/workarounds from my recent experience. Also, I added the instructions for Linux and Mac to the side panel, since Windows was already there.

## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by masclot** [comment]
@adhiamboperes PTAL

File: wiki/Bazel-Setup-Instructions-for-Linux.md [hunk 1]
```diff
@@ -25,10 +25,16 @@ Oppia Android also requires OpenJDK 8. The Bazel installation instructions above
 
    You can confirm that this is set up using the command `java -version`, which should result in three lines being printed out with the first one showing "openjdk version "1.8.0_292".
 
+> **Tip:** Newer linux distributions may not have openjdk-8 anymore. This has been reported for Debian and Ubuntu. If this is your case, **openjdk-11** it's been tested as a viable alternative.
```
Comment (adhiamboperes) [inline wiki/Bazel-Setup-Instructions-for-Linux.md]:
```suggestion
> **Tip:** Newer linux distributions may not have openjdk-8 anymore. This has been reported for Debian and Ubuntu. If this is your case, **openjdk-11** has been tested as a viable alternative.
```

**Review by adhiamboperes — APPROVED** [review]
Thanks @masclot. I have left a grammar correction, but otherwise LGTM.

File: wiki/Bazel-Setup-Instructions-for-Linux.md (hunk 1 above)
Comment (adhiamboperes) [reply wiki/Bazel-Setup-Instructions-for-Linux.md]: @masclot, could you please update this?

**Comment by masclot** [comment]
Comments addressed. @adhiamboperes PTAL

**Review by adhiamboperes — APPROVED** [review]
LGTM, thanks @masclot!

**Pull request merged**